viewer/packages/tools/src/HtmlOverlay/HtmlOverlayTool.ts

```diff
@@ -223,25 +223,27 @@ export class HtmlOverlayTool extends Cognite3DViewerToolBase {
 
     // Note! Must be part of DOM tree before we do getComputedStyle(), so add before check
     this.viewerDomElement.appendChild(htmlElement);
-    const style = getComputedStyle(htmlElement);
-    if (style.position !== 'absolute') {
-      this.viewerDomElement.removeChild(htmlElement);
-      throw new Error(`htmlElement style must have a position of absolute. but was '${style.position}'`);
-    }
-
-    const element: HtmlOverlayElement = {
-      position3D,
-      options,
-      state: {
-        position2D: new THREE.Vector2(),
-        width: -1,
-        height: -1,
-        visible: true
+    setTimeout(() => {
+      const style = getComputedStyle(htmlElement);
+      if (!style.position || style.position !== 'absolute') {
+        this.viewerDomElement.removeChild(htmlElement);
+        throw new Error(`htmlElement style must have a position of absolute. but was '${style.position}'`);
       }
-    };
-    this._htmlOverlays.set(htmlElement, element);
 
-    this.scheduleUpdate();
+      const element: HtmlOverlayElement = {
+        position3D,
+        options,
+        state: {
+          position2D: new THREE.Vector2(),
+          width: -1,
+          height: -1,
+          visible: true
+        }
+      };
+      this._htmlOverlays.set(htmlElement, element);
+
+      this.scheduleUpdate();
+    }, 50);
```

Comment:
this magic number 50 is to guarantee that it will be executed only after the html style is computed? Maybe I suggest to add some comments to explain this number or add as a constant. But it is more up to you IMO.